packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -59,6 +67,16 @@ Before you start using analytics, you need to add platform specific configuratio
 Every flavor represents a separate Firebase project that will be used for app tracking. For each flavor, based on the targeted platforms you'll have to download the [configuration files][firebase_configs_lnk] and place them in the appropriate location mentioned above.
 
 {{/analytics}}
+### Http client
```

Comment:
@RomanovaPrime, from what I see you are trying to explain in details all counter_feature specifics, which is not the goal of that readme file. Please consider this file as generic as possible. We need to answer the following questions in the most generic way.
1. What we can achieve with that HTTP client?
2. How we can achieve it?

Also please leave a links to any useful resources such as [json_annotation](https://pub.dev/packages/json_annotation) etc.

By reading this section of the documentation, everybody who is not familiar with this HTTP client should understands the basic principles. For more details they can follow the links

Reply:
I tried to make it more clear. Hope it is better now.